lib/ProductOpener/Food.pm

```diff
@@ -3669,6 +3740,34 @@ sub compute_nutrient_levels($) {
 	$product_ref->{nutrient_levels} = {};
 	
 	return if ($product_ref->{categories} eq '');	# need categories hierarchy in order to identify drinks
+		
+	# do not compute a score for dehydrated products to be rehydrated (e.g. dried soups, powder milk)
+	# unless we have nutrition data for the prepared product
+	
+	my $prepared = "";
+	
+	if (has_tag($product_ref, "categories", "en:dried-products-to-be-rehydrated")) {
+	
+			if ((defined $product_ref->{nutriments}{"energy_prepared_100g"})) {
+				$prepared = '_prepared';
+			}
+			else {
+				return;
+			}
+	}
+	
+	
+	# do not compute a score for coffee, tea etc.
+	if (	(has_tag($product_ref, "categories", "en:alcoholic-beverages")) 
+		or	(has_tag($product_ref, "categories", "en:coffees"))
+		or	(has_tag($product_ref, "categories", "en:teas"))
+		or	(has_tag($product_ref, "categories", "en:teas"))
+		or	(has_tag($product_ref, "categories", "fr:levure"))
+		or	(has_tag($product_ref, "categories", "fr:levures"))
```

Comment:
In fact the NutriScore is officially not applicable to a few categories like "honey", but it may be more a policy issue than something else. That's why I left the nutrient levels for some categories that are exempted from NutriScore. I will move both to the Config_off.pm file

Reply:
Moved to Config_off.pm